src/features/harborView/forms/harborForm/HarborEditForm.tsx

```diff
@@ -141,57 +140,6 @@ const HarborEditForm = ({ harborId, onCancel, onSubmit, refetchQueries }: Props)
             />
           </Section>
 
-          <Section>
```

Comment:
Would it make sense to put this behind a feature flag?

Reply:
I no because it has no value in local dev env either.

Reply:
It's just a shame if this feature needs to be reintroduced in the future and the assigned dev is unaware the implementation already existed. 😅